pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -445,6 +451,22 @@ message LinuxContainerSecurityContext {
     // Capabilities to add or drop.
     optional Capability capabilities = 1;
     // If set, run container in privileged mode.
+    // Privileged mode is incompatible with the following options. If
+    // privileged is set, the following features MAY have no effect:
+    // 1. capabilities
+    // 2. selinux_options
+    // 4. seccomp
+    // 5. apparmor
+    //
+    // Privileged mode implies the following specific options are applied:
+    // 1. All capabilities are added.
+    // 2. Sensitive paths, such as kernel module paths within sysfs, are not masked.
+    // 3. Any sysfs and procfs mounts are mounted RW.
+    // 4. Apparmor confinement is not applied.
+    // 5. Seccomp restrictions are not applied.
+    // 6. The device cgroup does not restrict access to any devices.
+    // 7. All devices from the host's /dev are available within the container.
+    // 8. SELinux restrictions are not applied (e.g. label=disabled).
```

Comment:
I would love to scope it down to something a little more sane if we can.

It's unfortunate that docker makes the level of granularity impossible. While OCI/runc supports each of these options at a granular level, docker does not provide api options for about half of these (the /dev bit, path masking, device cgroup tweaks, replicating the exact semantics of the sys/cgroup mounts).

Because of that, the only way to break it down is to also break the public API and make privileged to somewhat less, or to modify docker to have more knobs and wait for all supported versions to have them.

Neither of those options sounds great.